### PR TITLE
Remove info from SQS message and commit info when create it && Support CPU render

### PR DESCRIPTION
**Pull request by jiazhentim**
Decrease the message size by remove info from message
Support CPU rendering

### CODE REVIEW

**Comment by jiazhentim** [comment]
OK, I will do that @macrintr

File: inference/fold_detection.py [hunk 1]
```diff
@@ -78,7 +78,11 @@ def print_run(diff, n_tasks):
   start = time()
   # wait 
   n = len(batch)
+<<<<<<< HEAD
```
Comment (nkemnitz) [inline inference/fold_detection.py]: Resolve merge conflict

File: inference/docker/Dockerfile.cpu [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  && apt-get install -y python3-pip \
+  && apt-get install -y --no-install-recommends \
+      libgtk2.0-dev \
+  && pip3 install --no-cache-dir -r requirements.txt \
+  # Register SEAMLeSS
+  && python3 setup.py develop \
+  # Cleanup apt
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* 
+RUN pip3 install imageio==2.4.1
```
Comment (nkemnitz) [inline inference/docker/Dockerfile.cpu]: That one is part of the requirements.txt

File: inference/aligner.py [hunk 3]
```diff
@@ -536,6 +536,36 @@ def compute_field_chunk(self, model_path, src_cv, tgt_cv, src_z, tgt_z, bbox, mi
 
     return field
 
+  def perdict_image(self, cm, model_path, src_cv, dst_cv, z, mip, bbox,
```
Comment (nkemnitz) [inline inference/aligner.py]: Typo in function name

File: inference/aligner.py [hunk 4]
```diff
@@ -129,7 +129,7 @@ def adjust_bbox(self, bbox, dis):
   # IO methods #
   ##############
 
-  def get_model_archive(self, model_path):
+  def get_model_archive(self, model_path, readonly=1):
```
Comment (nkemnitz) [inline inference/aligner.py]: Use bool for flags

File: inference/docker/deploy_cpu.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    run: seamless
+  name: seamless
+spec:
```
Comment (nkemnitz) [inline inference/docker/deploy_cpu.yaml]: Include `dnsPolicy` setting from deploy.yml

File: inference/aligner.py [hunk 6]
```diff
@@ -1146,24 +1178,24 @@ def chunkwise(patch_bbox):
...
-      field = vector_vote(fields, T=T)
-      field = field.data.cpu().numpy() 
-      self.save_field(write_F_cv, z, field, bbox, mip, relative=False, as_int16=as_int16)
+          src_z = z
+          tgt_z = src_z - z_offset
+          print("calc res for src {} and tgt {}".format(src_z, tgt_z))
+          f = self.compute_field_chunk(model_path, src_cv, tgt_cv, src_z,
+                                       tgt_z, bbox, mip, pad)
+          #print("--------f shape is ---", f.shape)
+          fields.append(f.cpu().data)
```
Comment (nkemnitz) [inline inference/aligner.py]: `compute_field_chunk` now returns a numpy array, I believe.

**Pull request merged**